plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -688,9 +698,6 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
         addCodeModernizeUI(true)
         val maybeUnknownReason = reason ?: message("codemodernizer.notification.warn.invalid_project.description.reason.unknown")
         codeModernizerBottomWindowPanelManager.setProjectInvalidUI(maybeUnknownReason)
-        ApplicationManager.getApplication().invokeLater {
-            runInEdt { ValidationErrorDialog.create(maybeUnknownReason) }
```

Comment:
This was repetitive -- we already have a notification showing the exact same info and it pops up at the same time.

Reply:
Can you add screenshots to the PR with these changes to clearly show the changes (before and after). Makes buy in easier.

Reply:
See above